frontend/app/compontens/navbar.tsx

```diff
@@ -1,8 +1,11 @@
 'use client'
 import Link from 'next/link'
 import { LuGithub, LuClipboardCheck } from 'react-icons/lu'
+import getConfig from '../../next.config.cjs'
```

Comment:
Please use absolute imports with `@`. This will make refactorings easier.

`import getConfig from '@/next.config.cjs'`